src/error/metadata/suggestion.rs

```diff
@@ -108,6 +110,9 @@ RunComposition => {
 UseFederatedGraph => {
                 "Try running the command on a valid federated graph, or use the appropriate `rover graph` command instead of `rover subgraph`.".to_string()
             }
+            UseContractVariant => {
+                "Try running the command on a valid contract graph.".to_string()
```

Comment:
right now, the full error text looks like this:

```console
$ rover contract describe averys-supercloud
Fetching description for configuration of averys-supercloud@current using credentials from the default profile.

error[E039]: The variant `averys-supercloud@current` is a non-contract variant. This operation is only possible for contract variants.
        Try running the command on a valid contract graph.
```

In the error message it's referred to as a "contract variant", but in the suggestion it's referred to as a "contract graph". I think in our quick zoom you had said you wanted to use "contract variant" here - is that correct?
```suggestion
                "Try running the command on a valid contract variant.".to_string()
```

Reply:
Yep, looks like I missed a spot/used the wrong wording here, will change to "variant".